cli/index.ts

```diff
@@ -468,3 +471,13 @@ function getObjectOrJSON(options) {
       return {};
   }
 }
+
+function notifyUpdateCliVersion() {
+  const pkg = require('./package.json');
+  const notifier = updateNotifier({
+    pkg,
+    updateCheckInterval: 0,
+    shouldNotifyInNpmScript: true,
+  });
+  notifier.notify();
```

Comment:
Can we actually modify the message to add Changelog link?

Looks like this package supports message customization (https://github.com/yeoman/update-notifier#notifiernotifyoptions):

```
notifier.notify({message: 'Run `{updateCommand}` to update.\nChangelog: https://github.com/Redocly/redoc/releases/tag/{latestVersion}'});
```